site/content/en/docs/formats/ava_action.md

```diff
@@ -0,0 +1,122 @@
+---
+title: 'AVA action'
+linkTitle: 'AVA action'
+description: ''
+---
+
+## Format specification
+
+The AVA action format specification is available
+[here](https://arxiv.org/pdf/1705.08421.pdf).
+
+The dataset has annotations for recognizing an action per instance from video frames
+like visual tracking task. Specifically, the AVA action dataset contains frame indices,
+bounding box cooridnates, actions, and tracking ids in the annotation file. The action
+categories are described in `ava_action_list_v2.2.pbtxt`. For the ease use for object
+detection, the AVA action dataset provides the bounding box proposals from `Faster R-CNN`.
+
+Supported task / format:
+- Object detection - `ava`
+
+Supported annotation types:
+- `Bbox` (detection)
+
+## Import AVA action dataset
+
+The AVA action dataset is available for free download
+[here](https://research.google.com/ava/download.html#ava_actions_download).
+
+A Datumaro project with a AVA action source can be created in the following way:
+
+``` bash
+datum create
+datum import --format ava <path/to/dataset>
+```
+
+It is possible to specify project name and project directory. Run
+`datum create --help` for more information.
+
+The AVA action dataset directory should have the following structure:
+
+<!--lint disable fenced-code-flag-->
+```
+└─ Dataset/Category
```

Comment:
Just `Dataset/` ?

Reply:
Yes, that is mvtec indeed.

Reply:
Does AVA format also has its category name at the top level directory?

Reply:
I am sorry that I misunderstood. I will correct this.